### PR TITLE
Fix note to be nice

### PR DESCRIPTION
**Pull request by Geczy**

### CODE REVIEW

File: apps/calculator.py [hunk 1]
```diff
@@ -105,19 +105,20 @@ def venmo_message_maker(description,request,my_dic,tip_perc,tax_perc,fee_part,ti
         amount = request[key] # total requested dollars
 
         # statement construction
-        statement = f'Hi {key}! Food '
+        # ﹪ is required instead of % because of a bug in venmo note
+        statement = f'Waaasssuup {key}\nTotal '
```
Comment (pomkos) [inline apps/calculator.py]: not happening lmao

File: apps/calculator.py [hunk 2]
```diff
@@ -105,19 +105,20 @@ def venmo_message_maker(description,request,my_dic,tip_perc,tax_perc,fee_part,ti
...
+            statement += f', tip was {round(tip_perc*100,2)}﹪'
         if tax > 0.0:
-            statement += f', tax was {round(tax_perc*100,2)}{urllib.parse.quote("﹪")}'
+            statement += f', tax was {round(tax_perc*100,2)}﹪'
         if misc_fees > 0.0:
             statement += f', fees were ${round(fee_part,2)}'
 
-        statement += f'.%0AMade with {urllib.parse.quote("❤️")} at payme.peti.work' # %0A creates a new line
-        statement = statement.replace(' ','%20') # replace spaces for url parameter
+        statement += f'.\n\nMade with ❤️ at payme.peti.work' # %0A creates a new line
```
Comment (pomkos) [inline apps/calculator.py]: This didn't work in url.

**Review by pomkos — CHANGES_REQUESTED** [review]

File: apps/calculator.py [hunk 3]
```diff
@@ -117,7 +118,9 @@ def venmo_message_maker(description,request,my_dic,tip_perc,tax_perc,fee_part,ti
         if misc_fees > 0.0:
             statement += f', fees were ${round(fee_part,2)}'
 
-        statement += f'.\n\nMade with ❤️ at payme.peti.work' # %0A creates a new line
+        statement += f'''.
```
Comment (pomkos) [inline apps/calculator.py:121]: Why so many newlines

Comment (Geczy) [reply apps/calculator.py:121]: its more professional

**Review by pomkos — APPROVED** [review]

**Pull request merged**